google/cloud/sql/v1/sql_operations_connection.h

```diff
@@ -67,8 +68,8 @@ class SqlOperationsServiceConnection {
   virtual StatusOr<google::cloud::sql::v1::Operation> Get(
       google::cloud::sql::v1::SqlOperationsGetRequest const& request);
 
-  virtual StatusOr<google::cloud::sql::v1::OperationsListResponse> List(
-      google::cloud::sql::v1::SqlOperationsListRequest const& request);
+  virtual StreamRange<google::cloud::sql::v1::Operation> List(
+      google::cloud::sql::v1::SqlOperationsListRequest request);
```

Comment:
Aside: the change from `T const&` to `T` is fairly expensive.  I assume we cannot change it due to backwards compatibility reasons.  Do we have a plan to fix this?